package.json

```diff
@@ -29,13 +29,15 @@
     "bignumber.js": "^9.0.0",
     "canonicalize": "^1.0.1",
     "ethers": "^5.0.19",
+    "isomorphic-fetch": "^3.0.0",
```

Comment:
Would this need using for all uses of `fetch` now? Could be good to add a design note in README?

Reply:
Done